global-test.properties

```diff
@@ -16,8 +16,8 @@ intersmash.bootable.jar.image=registry.access.redhat.com/ubi8/openjdk-17
 intersmash.mysql.image=quay.io/centos7/mysql-80-centos7:centos7
 intersmash.postgresql.image=quay.io/centos7/postgresql-13-centos7:centos7
 
-intersmash.wildfly.image=quay.io/wildfly/wildfly-s2i-jdk17:latest
-intersmash.wildfly.runtime.image=quay.io/wildfly/wildfly-runtime-jdk17:latest
+intersmash.wildfly.image=quay.io/wildfly/wildfly-s2i:latest
+intersmash.wildfly.runtime.image=quay.io/wildfly/wildfly-runtime:latest
```

Comment:
@fabiobrz that's new for me: no more JDK ?

Reply:
Hi @tommaso-borgato. Exactly, this is the new image that should be used, other ones would fail running the s2i build since with the latest plugin, which requires Maven 3.6.3+

Reply:
Tnks @fabiobrz